Integrations/Flashpoint/Flashpoint.yml

```diff
@@ -0,0 +1,413 @@
+      description: Title of the report.
+      type: string
+    - contextPath: Flashpoint.Report.updated_at
+      description: Last updated date of report.
+      type: string
+    description: Get a single report by its ID
+  - name: flashpoint-get-related-reports
+    arguments:
+    - name: report_id
+      required: true
+      description: Search related report by report id.
+    outputs:
+    - contextPath: Flashpoint.Reports
+      description: Display list of related report based on report fpid.
+    description: Get related reports for a given report id
+  - name: flashpoint-get-single-event
+    arguments:
+    - name: event_id
+      required: true
+      description: The UUID or FPID that identifies a particular event.
+    outputs:
+    - contextPath: Flashpoint.event.date
+      description: Date of event triggered.
+      type: string
+    - contextPath: Flashpoint.event.event_creator_email
+      description: Event creator email.
+      type: string
+    - contextPath: Flashpoint.event.href
+      description: Display event reference.
+    - contextPath: Flashpoint.event.tag
+      description: Display event tag.
+    - contextPath: Flashpoint.event.event_id
+      description: Display event id (event fpid)
+      type: string
+    description: For getting single event
+  - name: flashpoint-get-events
+    arguments:
+    - name: time_period
+      description: Specified time period. Search events based on time period.
+    - name: report_fpid
+      description: Search events by report fpid.User can get report fpid from output of report-search or related-reports
+        commands and use it in this command to get events for specific Flashpoint report.
+    - name: limit
+      description: Specify limit of the record.
+      defaultValue: "10"
+    - name: attack_ids
+      description: Search events by attack ids. Multiple ids are acceptable using
+        comma separated values. attack_ids can be found in event information.
+    outputs:
+    - contextPath: Flashpoint.events
+      description: Display the list of multiple events.
+    description: Get all event details
+  - name: flashpoint-common-lookup
+    arguments:
+    - name: indicator
+      required: false
+      default: true
+      description: Specify indicator value like any domain, ip, email, url etc.
+      isArray: true
+    outputs:
+    - contextPath: DBotScore.Indicator
+      description: The indicator that was tested.
+      type: string
+    - contextPath: DBotScore.Score
+      description: The indicator score.
+      type: number
+    - contextPath: DBotScore.Type
+      description: The indicator type.
+      type: string
+    - contextPath: DBotScore.Vendor
+      description: The vendor used to calculate the score.
+      type: string
+    description: Lookup any type of indicator
+  - name: flashpoint-get-forum-details
+    arguments:
+    - name: forum_id
+      required: true
+      description: Specify forum id
```

Comment:
Add more description, where the user should take the forum id? Which command and which output?

Reply:
Added description